### PR TITLE
   operations in the environment module ,adding eco systems view the

### PR DESCRIPTION
**Pull request by Gethmi**
From the environment module we can add new Eco systems and new species. also admin can view the requests using the check status . users can track their own request .

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,7 +11,7 @@ LOG_CHANNEL=stack
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
-DB_DATABASE=laravel
+DB_DATABASE=laravelpro
```
Comment (yashodgayashan) [inline .env.example:14]: Please don't push passwords

**Review by yashodgayashan — COMMENTED** [review]
Please remove unnecessary spaces.

File: .env.example (hunk 1 above)
Comment (Gethmi) [reply .env.example:14]: will do

**Comment by yashodgayashan** [comment]
@Gethmi Please resolve conflicts and make the PR again

**Pull request closed without merging**